dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentAsDraftActionlet.java

```diff
@@ -78,15 +78,16 @@ public void executeAction(final WorkflowProcessor processor,
 
 			final Contentlet contentlet             = processor.getContentlet();
 			final User user                         = processor.getUser();
-			List<Category > categories        = null;
-			final List< Permission > permissions    =
-					this.permissionAPI.getPermissions(contentlet, false, true);
+			List<Category > categories              = null;
 
 			Logger.debug(this,
 					"Saving the content as draft of the contentlet: " + contentlet.getIdentifier());
 
 			contentlet.setProperty(Contentlet.WORKFLOW_IN_PROGRESS, Boolean.TRUE);
 			final ContentletDependencies contentletDependencies = processor.getContentletDependencies();
+			final List< Permission > permissions    = null != contentletDependencies && null != contentletDependencies.getPermissions()?
```

Comment:
I don't remember if we agreed with @wezell to support other actions, not just Save as Draft, do you?

Reply:
the method is one the ESContentletAPI, however we need to do this specific work here to make it work